test/models/outbound_webhook_test.rb

```diff
@@ -143,14 +143,16 @@ def assert_webhook_fired(&block)
         assert_request :post, "https://testing.com/deploys", to_return: polling_target, &block
       end
 
+      with_env OUTBOUND_WEBHOOK_POLL_PERIOD: '1'
```

Comment:
- test the polling work by expecting sleeps
 - add a `expects(:sleep).never` to the default before so we don't sleep forever when something goes wrong